docs/props-spread.md

```diff
@@ -2,4 +2,10 @@
 title: Props Spread
 ---
 
-You can't. Props spreading is a big source of unpredictability and performance regression (think `shouldComponentUpdate`). Our API prevents this. If you reaaaaally need it for binding to existing ReactJS components, see [this section](clone-element.md).
+There is currently no story around doing this in a type-safe manner.
```

Comment:
Maybe something like "There is currently no way to do this in a type-safe manner." - I'm not sure the "no story" phrase translates well.